rollup.config.mjs

```diff
@@ -3,22 +3,22 @@ import commonjs from "@rollup/plugin-commonjs";
 import nodeResolve from "@rollup/plugin-node-resolve";
 import json from "@rollup/plugin-json";
 import nodePolyfills from "rollup-plugin-polyfill-node";
-import globals from "rollup-plugin-node-globals";
 
 // Debugging tools
 import { visualizer } from "rollup-plugin-visualizer";
 
-import pickFromPackageJson from "./scripts/pick-from-package-json";
-import { version } from "./package.json";
+import pickFromPackageJson from "./scripts/pick-from-package-json.mjs";
+import packageJson from "./package.json" with { type: "json" };
```

Comment:
why change this one? does the old `import { version } from "./package.json";` no longer work? would it work if we used `import { version } from "./package.json" with { type: "json" };` ?